lib/shape.mli

```diff
@@ -2,9 +2,9 @@ type shape
 type shapes = shape list
 
```

Comment:
After adding a `color` type, you would also need to add it here and change the function signatures.